apps/asap-server/package.json

```diff
@@ -8,14 +8,15 @@
   "scripts": {
     "build:babel": "../../scripts/build-babel.sh build",
     "watch:babel": "../../scripts/build-babel.sh watch",
-    "start:server": "tsnd --transpile-only ./src/server.ts",
+    "start": "node-dev ./src/server.ts",
```

Comment:
I renamed this to `start` for consistency with the other apps (and because `server` is already in the package name so it should be clear). In the root it's of course still aliased `start:backend`